Assets/MRTK/Core/Services/MixedRealityToolkit.cs

```diff
@@ -329,7 +329,10 @@ private void RemoveCurrentProfile(MixedRealityToolkitConfigurationProfile profil
             if (IsInitialized)
             {
                 DebugUtilities.LogVerboseFormat("Unregistered service of type {0} was an initialized service, disabling and destroying it", typeof(T));
-                serviceInstance.Disable();
+                if (activeProfile != null && Application.IsPlaying(activeProfile))
```

Comment:
to be clear, resetting using the current profile (or no profile) will do a full teardown and reinitialize. if changing profiles, there is no teardown.

correct?

Reply:
ignore me.... i misread this :D